WTWSMS/history/characters/WTWSMS_enlil.txt

```diff
@@ -7612,6 +7625,17 @@
 		birth = yes
 	}
 	580.1.1={
+=======
+	name = "Luctesgus"
+	# Luctesgus Viridothreus
+	dynasty = 1042523
+	religion = "nicene"
+	culture = "astures"
+	530.1.1 = {
+		birth = yes
+	}
+	610.1.1 = {
+>>>>>>> master
 		death = yes
 	}
 }
```

Comment:
There is a conflict here, caused by the "cleaning" in the master branch. You will have to resolve this if you want to incorporate the "cleaning".